README.md

```diff
@@ -45,7 +45,8 @@ Please read [COMPILING.md](doc/COMPILING/COMPILING.md) - it covers general infor
 
 We also have the following build guides:
 * Building on Windows with `MSYS2` at [COMPILING-MSYS.md](doc/COMPILING/COMPILING-MSYS.md)
-* Building on Windows with `vcpkg` at [COMPILING-VS-VCPKG.md](doc/COMPILING/COMPILING-VS-VCPKG.md)
+<!-- Instructions temporarily deprecated due to fatal error, tracked as issue 15087 at https://github.com/microsoft/vcpkg/ -->
```

Comment:
There could be all kinds of errors in third party tools (e.g. in vcpkg), but there is no need to hide/remove instruction which is perfectly fine because of any possible errors.

Reply:
Is it "perfectly fine" if the instructions don't work, though?

Reply:
Yes, our instructions are perfectly fine - they worked before the issue hit vcpkg and they will work after that vcpkg issue is resolved.

Reply:
I am not against updating the instruction, but it would be better to change it in such generic way that we won't need to change it again once a different issue hits vcpkg - e.g. refer to specific version of vcpkg that does not have any known issues when building CDDA, totally ignore any issues that aren't directly related to CDDA (and assume there are would be no error when installing vcpkg, building CDDA), assume there are would be all kinds of errors and ask reader to seek help for possible solutions in vcpkg repo - something like that.

Reply:
> totally ignore any issues that aren't directly related to CDDA (and assume there are would be no error when installing vcpkg, building CDDA)

So, while I understand that we can't possibly keep our instructions up-to-date with every single change, I ran into this error while attempting to clean them up, and it's a hard stop. I don't know if there *is* a version of the yasm subcomponent that works -- if there is, I sure couldn't find it, and knowing that something is fatally wrong, leaving the instructions as they are doesn't exactly sit well with me.

I admit I don't have a perfect solution, but "ignore the error and assume everything works" doesn't feel right -- not least of which because I can't verify the downstream instructions work.